recipes-kernel/linux/files/bcm2712-raspberrypi5-wifi.dtso

```diff
@@ -0,0 +1,127 @@
+/*
+ * Copyright (c) 2024 EPAM systems
+ *
+ * SPDX-License-Identifier: Apache-2.0
+*/
+/dts-v1/;
+/plugin/;
+
+&passthrough {
+	pinctrl@107d504100 {
```

Comment:
pinctrl looks like a problem, unfortunately :( We can't jsut passthrough it, as it's shared resource. All current HW working with assumption pinctrl is configured already by RPI5 bootloader or u-boot.

Reply:
UPD: Not sure how to proceed :(

Reply:
Yeah, in cases like this we are considering using SCMI. This is the best approach, but it is PITA to implement. 

Another approach is to configure pinctrl in bootloader/dom0 so guest can use static configuration. Of course, it will not work if dynamic configuration is required.